src/components/__stories__/constants/pinning.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import type {ColumnDef} from '@tanstack/react-table';
+
+import {ColumnPinningCell, ColumnPinningHeaderCell} from '../cells/ColumnPinningCell';
+import type {Item} from '../types';
+
+export const columns: ColumnDef<Item>[] = [
```

Comment:
Let's move `columns` to `ColumnPinningDemo.tsx`. It's not use anywhere else anyway.